static/templates/message_body.handlebars

```diff
@@ -1,23 +1,21 @@
 <div class="message_top_line">
-    <span class="message_sender{{^status_message}} sender_info_hover{{/status_message}} no-select">
+    {{#unless status_message}}
+    <span class="message_sender sender_info_hover no-select">
 
         {{#if include_sender}}
         {{partial "message_avatar"}}
         {{/if}}
 
-        {{#if status_message}}
-        {{partial "me_message"}}
-        {{/if}}
-
-        {{#if include_sender}}
-            {{#unless status_message}}
-                <span class="sender_name auto-select" role="button" tabindex="0">{{msg/sender_full_name}}</span>
-                {{#if sender_is_bot}}
-                <i class="zulip-icon bot" aria-label="{{t 'Bot' }}"></i>
-                {{/if}}
-            {{/unless}}
+        <span class="sender_name auto-select" role="button" tabindex="0">{{msg/sender_full_name}}</span>
+        {{#if sender_is_bot}}
+        <i class="zulip-icon bot" aria-label="{{t 'Bot' }}"></i>
```

Comment:
I think this block accidentally slipped outside the `include_sender` block?

Reply:
oops, yep